Sequences/MRML/vtkMRMLLinearTransformSequenceStorageNode.cxx

```diff
@@ -283,6 +283,7 @@ int vtkMRMLLinearTransformSequenceStorageNode::ReadSequenceFileTransforms(const
         }
         numberOfCreatedNodes++;
         transformsSequenceNode = newTransformsSequenceNode;
+        newTransformsSequenceNode->Delete();
```

Comment:
An other approach would be to update line 267 to use:

```
vtkSmartPointer<vtkMRMLSequenceNode> newTransformsSequenceNode;
```

instead of 

```
vtkMRMLSequenceNode* transformsSequenceNode = NULL;
```.

And `newTransformsSequenceNode = vtkSmartPointer<vtkMRMLSequenceNode>::New();` at link 281

Reply:
Do you mean update line 271 to use?:
```
vtkSmartPointer<vtkMRMLSequenceNode> newTransformsSequenceNode;
```
Anyways it doesn't seem to be working for me

Reply:
Yes.

Two changes:

line 271: `vtkSmartPointer<vtkMRMLSequenceNode> newTransformsSequenceNode;`
line 281: `newTransformsSequenceNode = vtkSmartPointer<vtkMRMLSequenceNode>::New();`

Reply:
Got it working, thanks! Pushed update